### ISSUE TITLE
Can't build with panic = "abort"

### ISSUE THREAD

**Issue by ye2020** [oschwald/maxminddb-rust#33]
Using this crate with the following option in `Cargo.toml` would cause a compile error
```toml
[profile.release]
panic = "abort"
```

```
~/rust/testmmdb$ cargo build --release
   Compiling cfg-if v0.1.10
   Compiling serde v1.0.115
   Compiling log v0.4.11
   Compiling maxminddb v0.14.0
error: the linked panic runtime `panic_unwind` is not compiled with this crate's panic strategy `abort`

error: aborting due to previous error

error: could not compile `maxminddb`.
```

Perhaps we can [remove this line](https://github.com/oschwald/maxminddb-rust/blob/85f1b3346d18e2eadd9600abfe26613337d25fc7/Cargo.toml#L23) ?

**Comment by oschwald** [comment]
Sorry for not responding, but I am considering it for the next release. I do want to figure out if removing those types will significantly impact anyone.

**Comment by fasterthanlime** [comment]
You probably want to remove these lines too https://github.com/oschwald/maxminddb-rust/blob/85f1b3346d18e2eadd9600abfe26613337d25fc7/src/maxminddb/lib.rs#L2-L3

**Comment by eddyb** [comment]
It shouldn't impact anyone, you should avoid specifying `crate_type` in general, looks like it was left over from pre-Rust-1.0 days.

We should've deprecated `dylib` a long time ago, and `staticlib` is only useful if you're building a standalone C library (with a C API, which you don't have, so it does nothing for you).

**Comment by oschwald** [comment]
Thanks for the feedback! I released 0.15.0 with this change.
